packages/grid/_modules_/grid/components/cell/GridRowCells.tsx

```diff
@@ -25,6 +25,7 @@ function applyCssClassRules(cellClassRules: GridCellClassRules, params: GridCell
 }
 
 interface RowCellsProps {
+  className?: string;
```

Comment:
```suggestion
  cellClassName?: string;
```

Reply:
why not use the generic term?

Reply:
I guess that comment answeres it https://github.com/mui-org/material-ui-x/pull/1660#discussion_r632513560